src/lesson2/Lesson2Ex11.java

```diff
@@ -0,0 +1,13 @@
+package lesson2;
+
+public class Lesson2Ex11 {
+    public static void main(String[] args) {
+        boolean doesSignificantWork = true;
```

Comment:
es popoxakannery scannerov mutq ara,
verjum el gone mi ban tpi imananq inch exav))

Reply:
chishtn asac es meki pahanjy yndhanrapes chem haskacel (((

Reply:
haskanum em inch a uzum, bayc chem haskanum inch anem

Reply:
mi hat uxarki xndiry estex:
copy ara pahanjy commentov gri asem

Reply:
Consider the following if statement, where doesSignificantWork ,
makesBreakthrough , and nobelPrizeCandidate are all boolean variables:
if (doesSignificantWork) {
if (makesBreakthrough)
nobelPrizeCandidate = true;
else nobelPrizeCandidate = false;
} else if (!doesSignificantWork)
nobelPrizeCandidate = false;
First, write a simpler if statement that is equivalent to this one. Then write a single
assignment statement that does the same thing.

Reply:
ha lov ok, esi esh bana mi xoraci :D

Reply:
ha indz el dur chekav :D